test/integration/src/test/java/io/pravega/test/integration/BatchClientWithAuthTest.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.integration;
+
+import io.pravega.client.ClientConfig;
+import io.pravega.client.stream.impl.DefaultCredentials;
+import io.pravega.controller.server.rpc.auth.StrongPasswordProcessor;
+import io.pravega.segmentstore.server.host.stat.AutoScalerConfig;
+import io.pravega.segmentstore.server.store.ServiceBuilder;
+import io.pravega.segmentstore.server.store.ServiceBuilderConfig;
+import io.pravega.segmentstore.server.store.ServiceConfig;
+import io.pravega.test.integration.utils.PasswordAuthHandlerInput;
+import io.pravega.test.integration.demo.ControllerWrapper;
+import org.junit.AfterClass;
+
+import java.io.File;
+import java.net.URI;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
+/**
+ * The tests in this class are intended to verify whether Batch Client works with a Pravega cluster
+ * that has "Auth" (short for authentication and authorization) enabled.
+ *
+ * This class inherits the tests of the parent class. Some of the test methods of the parent are reproduced here as
+ * handles, to enable running an individual test interactively (for debugging purposes).
+ */
+public class BatchClientWithAuthTest extends BatchClientTest {
+
```

Comment:
Do we also have a test that includes both `Auth` and `TLS`?

Reply:
We don't have a batch client test class that includes both `auth` and `TLS` or with only `TLS`. I just added one for `auth` in this PR. I didn't add test classes for TLS, as that change, although desirable, wouldn't be relevant for the current issue and PR. Or maybe it is: how do you know this will work in tandem with `TLS` otherwise. Just thinking out loud. 

What do you say?